templates/netlify/remix.init/globals.d.ts

```diff
@@ -0,0 +1,14 @@
+export {};
```

Comment:
We currently use this in our template, https://github.com/netlify/remix-edge-template/blob/main/globals.d.ts